zenoh/path.py

```diff
@@ -32,33 +36,29 @@ def is_valid(self, path):
             and not path.startswith('//')
 
     def is_absolute(self):
-        if self.path.startswith('/'):
+        if self._path.startswith('/'):
             return True
         return False
 
     def is_prefix(self, prefix):
-        return self.path.startswith(prefix)
-
-    def remove_prefix(self, prefix):
-        if self.is_prefix(prefix):
-            self.path = self.path[len(prefix):]
+        return self._path.startswith(prefix)
 
     def to_string(self):
-        return self.path
+        return self._path
 
     def __len__(self):
-        return len(self.path)
+        return len(self._path)
 
     def __eq__(self, second_path):
         if isinstance(second_path, self.__class__):
-            return self.path == second_path.path
+            return self._path == second_path_.path
```

Comment:
```suggestion
            return self._path == second_path._path
```
Typo: the _ should be added at the beginning of 'path' attribute. Not at the end of 'second_path'

Reply:
@JEnoch 
what a stupid mistake hahaha
fixed it